src/fhir.js

```diff
@@ -240,8 +240,10 @@ class FHIRObject {
     return dateOrIvl;
   }
 
-  _is(namespace, name) {
-    return this._typeHierarchy().some(t => t.namespace === namespace && t.name == name);
+  _is(typeSpecifier) {
+    return this._typeHierarchy().some(
+      (t) => t.type === typeSpecifier.type && t.name == typeSpecifier.name
```

Comment:
This code basically assumes that the `typeSpecifier` is a `NamedTypeSpecifier` (because I think that's the only one that has a `name` property).  I think that's OK because for this data model, the only things you would call `_is` on are `NamedTypeSpecifier` types.  We don't use this class (`FHIRObject`) to represent lists or intervals or choices, so I think we're good here.

(I mention this because I think you asked about it in Slack).